CONTRIBUTING.md

```diff
@@ -2,7 +2,28 @@
 
 Enhancements and fixes are always welcome. Feel free to contribute to any of the Issues not already assigned to another person.
 
-* `pip install -r dev_requirements.txt`
-* `cd tests`
-* `pytest`
-* New code should have unit tests coverage
+1. Install [Python 3](https://www.python.org/downloads/) and pip
+1. Install [dotent 5.0 runtime](https://dotnet.microsoft.com/download/dotnet/5.0/runtime)
+1. Clone this repository
+1. In the repository modify the `configuration.local.json`
+1. Set `"PythonAndGUploadInstalled": true`
+1. Open the command line
+
+```
+> cd peloton-to-garmin
+> cd python
+> pip install -r requirements.txt
+> cd ../
+> dotnet restore
+> dotnet build
+> dotnet run ./PelotonToGarminConsole/PelotonToGarminConsole.csproj
```

Comment:
Have you moved the directories a bit?  I had to launch it like this:
```
$ dotnet run --project ./src/PelotonToGarminConsole/PelotonToGarminConsole.csproj
```

And that only worked once I added my configuration to this directory:
```
cp configuration.example.json src/PelotonToGarminConsole/
```

Reply:
Contributing.md seems to be a little out of date, I'll update it in the next commit. I think the [wiki page](https://github.com/philosowaffle/peloton-to-garmin/wiki/Quick-Start#source) documentation is up to date though.

Reply:
Or, based on the CI failures, I might have messed up something in the rebase, will take a closer look.

Reply:
Yeah I saw the CI rereferring to different paths too, hence my confusion.  I think you should just modify the contributing.md to point people at the wiki page for info.

Reply:
Yup, looks like I busted the PR CI job when I rebased. Will need to fix it after work.